simulator/examples/analog-clock.rs

```diff
@@ -46,7 +46,7 @@ fn draw_face() -> impl Iterator<Item = Pixel<BinaryColor>> {
     // Use the circle macro to create the outer face
     let face = egcircle!(
         center = CENTER,
-        radius = SIZE,
+        diameter = DISP_SIZE as u32,
```

Comment:
This should be `SIZE * 2` to retain the old look.

```suggestion
        diameter = SIZE * 2,
```

Reply:
OK. Done.